views/view-aquilon-services.php

```diff
@@ -0,0 +1,59 @@
+<?php
+require("header.php"); // Important includes
+
+// Configuration
+$AQUILON_URL = $CONFIG['URL']['AQUILON'];
+$config = Array(
+    "clickable" => true,
+);
+
+$REPORTS = Array(
+    "Servers" => "$AQUILON_URL/cgi-bin/report/report_service_servers",
+    "Clients" => "$AQUILON_URL/cgi-bin/report/report_service_clients",
+);
+
+// Generates main array
+$results = Array();
+$results['config'] = $config;
+
+// Gets notes for nodes
+$all_notes = Array();
+$notes = $SQL->query("select name, note from notes");
+if ($notes and $notes->num_rows) {
+    while ($note = $notes->fetch_assoc()) {
```

Comment:
an assignemnt in a while loop condition?

Reply:
yep, fairly standard DB access stuff, keep looping until we run out of rows.